etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/orders/OrderConverter.java

```diff
@@ -9,4 +11,6 @@ public interface OrderConverter {
     Order<?> convertMetadataToOmlOrder(Order<?> order);
 
     Order<?> addContactSectionToPatientResource(Order<?> order);
+
+    OperationOutcome extractPublicMetadataToOperationOutcome(PartnerMetadata metadata);
```

Comment:
Sadly, I think I need to request changes.  `OrderConverter` is supposed to be FHIR-library agnostic, but `org.hl7.fhir.r4.model.OperationOutcome` is a HAPI FHIR class.  Instead, I think that we should model this off how we did `Order` and `HapiOrder`.  We could, for example, have a `FhirMetadata<T>` interface (similar to `Order`) and then have an implementing `HapiFhirMetadata<OperationOutcome>` class.

I'm happy to pair with you on this when I get online tomorrow.